src/main/resources/application.yaml

```
@@ -37,6 +37,7 @@ spring:
           name: discodeit
 
 
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)